internal/analytics/client/async/client.go

```diff
@@ -93,6 +93,10 @@ func (a *Client) Wait() {
 }
 
 func (a *Client) sendEvent(category, action, label string, dims ...*dimensions.Values) error {
+	if condition.InUnitTest() {
+		return nil
+	}
```

Comment:
It may be worth making this more general with `if a.svcModel == nil {`

Reply:
Also, to note (I feel like I might have mentioned this before), I would normally leave this as a nit message and approve the PR so that, if you don't care to make the change, you can merge right away. Though, I've seen your PRs with auto-merging, so I am hesitant and mark my nits with "request changes". Just be aware that changes are not necessarily expected when the only comments made are nits.

Reply:
Fair enough. I've also annotated it with a comment.